pages/index.js

```diff
@@ -1,8 +1,56 @@
 import styled from "styled-components";
 import SpotCard from "../components/SpotCard";
-import spots from "../helpers/mockSpots";
+import CreateSpot from "../components/CreateSpot";
+import { Button } from "../components/Button";
+import { useState, useEffect } from "react";
 
 export default function Home() {
+  const [state, setState] = useState("hide");
```

Comment:
`state `and `setState` are not really descriptive. In addition, I would prefer to change the state in boolean value.  In this case, however, you would have to query for a boolean in line 49ff. and 69ff, not for the explicit string. 
```suggestion
  const [isShown, setIsShown] = useState(false);
```